tidy3d/components/scene.py

```diff
@@ -234,6 +234,8 @@ def intersecting_media(
             Object for which intersecting media are to be detected.
         structures : List[:class:`.AbstractMedium`]
             List of structures whose media will be tested.
+        zero_dim_ind : int
```

Comment:
technically in the docstrings for optional arguments (with defaults) we'd put the default in here like 

```
zero_dim_ind : int = None
```